### PR TITLE
Cleanup passthrough encoder and unit tests

### LINKED ISSUE #2035: Cleanup passthrough encoder unit tests
Part of #2000

### PR DESCRIPTION
**Pull request by oxtopus**
Fixes #2035

### CODE REVIEW

File: nupic/encoders/pass_through_encoder.py [hunk 1]
```diff
@@ -49,49 +48,48 @@ def __init__(self, n, w=None, name="pass_through", forced=False, verbosity=0):
...
-  ############################################################################
+
   def getBucketIndices(self, input):
     """ See method description in base.py """
     return [0]
 
-  ############################################################################
-  def encodeIntoArray(self, input, output):
+
+  def encodeIntoArray(self, inputVal, outputVal):
```
Comment (david-ragazzi) [inline nupic/encoders/pass_through_encoder.py]: @oxtopus Why the `Val` suffix? It seems `input` and `output` names are common everwhere in code.

File: tests/unit/nupic/encoders/pass_through_encoder_test.py [hunk 2]
```diff
@@ -64,9 +63,9 @@ def testEncodeBitArray(self):
     bitmap[3] = 1
     bitmap[5] = 1
     out = e.encode(bitmap)
-    sum_expected = sum(bitmap)
-    sum_real = out.sum()
-    self.assertEqual(sum_real, sum_expected)
+    expectedSum = sum(bitmap)
```
Comment (david-ragazzi) [inline tests/unit/nupic/encoders/pass_through_encoder_test.py]: much better..

File: nupic/encoders/pass_through_encoder.py (hunk 1 above)
Comment (chetan51) [inline nupic/encoders/pass_through_encoder.py]: I believe `input` and `output` are keywords in Python.

**Comment by chetan51** [comment]
:+1:

**Pull request merged**